.gitignore

```diff
@@ -4,8 +4,8 @@ package-lock.json
 
 # build output
 .nyc_output
-/**/lib
-/**/build
+lib
+build
```

Comment:
May want to make sure this doesn't ignore the `tools/build` folder. I've had that happen to me before...

Reply:
Oh wow, never thought about this. luckily you caught this earlier :) 

![image](https://user-images.githubusercontent.com/11051042/136087794-1acecba5-3f4e-43f9-8340-16ef14d6c115.png)